benchmark/benchmarks-assembly.jl

```diff
@@ -18,29 +18,31 @@ for spatial_dim ∈ 1:3
 
         # Nodal interpolation tests
         for order ∈ 1:2, ip_type ∈ [Lagrange, Serendipity]
-            ip = ip_type{spatial_dim, ref_type, order}()
+            ip_type == Serendipity && (order < 2 || ref_type ∉ (RefQuadrilateral, RefHexahedron)) && continue
```

Comment:
Can you open an issue after we merge this with a small description of the issue here? Because I think this currently only tests Lagrange order 1 on Quad and Hex (which seems like a weird restriction)

Reply:
I think it tests Lagrange order 1 on Quad and Hex, the idea is that these are the two Serendipity implemented

Reply:
Oh, right, not sure how I missed this. :D